account_financial_report_webkit_xls/report/partners_balance_xls.py

```diff
@@ -376,7 +376,7 @@ def generate_xls_report(self, _p, _xs, data, objects, wb):
                              regular_cell_style_decimal),
                         ]
                     else:
-                        debit_col = 3
+                        debit_col = 4
```

Comment:
Yeah, but I wonder why the differentiation and the variable debit_col for at the end, getting the same value.